net/modemmanager/files/modemmanager.proto

```diff
@@ -393,17 +409,30 @@ proto_modemmanager_setup() {
 		cliauth="${cliauth}${cliauth:+|}$auth"
 	done
 
+
+	mmcli --modem="${device}" --timeout 120 --enable || {
+		proto_notify_error "${interface}" MM_MODEM_DISABLED
+		return 1
+	}
+
+	mmcli --modem="${device}" --3gpp-register-home || {
```

Comment:
Some people use SIM cards while roaming. In mainland China, using a Hong Kong SIM is a way to get out of the firewall. Also, some virtual-only mobile operators (MNVOs) do not have a home network. So, please don't hard-code the requirement to register in the home network.